common/src/Model/TagAttribute.cpp

```diff
@@ -1,39 +1,30 @@
 /*
  Copyright (C) 2010-2017 Kristian Duske
- 
+
  This file is part of TrenchBroom.
- 
+
  TrenchBroom is free software: you can redistribute it and/or modify
  it under the terms of the GNU General Public License as published by
  the Free Software Foundation, either version 3 of the License, or
  (at your option) any later version.
- 
+
  TrenchBroom is distributed in the hope that it will be useful,
  but WITHOUT ANY WARRANTY; without even the implied warranty of
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  GNU General Public License for more details.
- 
+
  You should have received a copy of the GNU General Public License
  along with TrenchBroom. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#ifndef Tutorial_h
-#define Tutorial_h
+#include "TagAttribute.h"
 
-#include "Color.h"
-#include "Assets/EntityDefinition.h"
-#include "Model/BrushContentType.h"
+#include "Model/Tag.h"
 
 namespace TrenchBroom {
     namespace Model {
-        namespace Tutorial {
-            extern const AttributeValue Classname;
-            extern const AttributeName  Message;
-            
-            Assets::EntityDefinition* createTutorialEntityDefinition();
-            BrushContentType createTutorialBrushContentType(BrushContentType::FlagType flag);
+        namespace TagAttributes {
```

Comment:
Suggest putting this in Tag.cpp, due to the high cost in compilation time of adding .cpp files

Reply:
Tag.h / cpp contains the general concept of tagging and shouldn't know anything about the context in which tags are used (e.g. references to other Model classes such as World). I know that in this PR, the references are still there, but they are already removed in the other PR.

So this is domain specific stuff, whereas Tag is a general concept. That's why I wanted to separate them.